src/seer/automation/autofix/autofix_agent.py

```diff
@@ -63,6 +69,49 @@ def _check_prompt_for_help(self, run_config: RunConfig):
                 "You're taking a while. If you need help, ask me a concrete question using the tool provided."
             )
 
+    def get_completion(self, run_config: RunConfig):
```

Comment:
Little comment for cleanliness so folks grok right away why this method is special for the autofix agent

```suggestion
    def get_completion(self, run_config: RunConfig):
    """
    Streams the preliminary output to the current step and only returns when output is complete
    """
```